pennylane_qiskit/converter.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+QuanctumCircuit converter module
+================================
+
+.. currentmodule:: pennylane_qiskit.load
+
+This module contains functions for converting Qiskit QuantumCirducit objects
+PennyLane templates.
+"""
+import warnings
+
+import numpy as np
+
+from qiskit import QuantumCircuit
+from qiskit.circuit import Parameter
+from qiskit.exceptions import QiskitError
+
+import pennylane.ops.qubit as pennylane_ops
+from pennylane_qiskit.qiskit_device import QISKIT_OPERATION_MAP
+
+# pylint: disable=too-many-instance-attributes
+
+inv_map = {v.__name__: k for k, v in QISKIT_OPERATION_MAP.items()}
+
+
+class WiresError(Exception):
+    """Exception raised by  the pennylane_qiskit.converter when it encounters an illegal
+    operation in the quantum circuit.
+    """
+
+
+def check_parameter_bound(param):
+    """Utility function determining if a certain parameter in a QuantumCircuit has
+    been bound.
+
+    Args:
+        param: the parameter to be checked
+
+    Returns:
+        param: the parameter after the check
+    """
+    if isinstance(param, Parameter):
+        raise ValueError("The parameter {} was not bound correctly.".format(param))
+    return param
+
+
+def load(quantum_circuit: QuantumCircuit):
+    """Returns a PennyLane template created based on the input QuantumCircuit or QASM string.
+    Warnings are created for each of the QuantumCircuit instructions that were
+    not incorporated in the PennyLane template.
+
+    Args:
+        quantum_circuit (qiskit.QuantumCircuit): the QuantumCircuit to be converted
+
+    Returns:
+        function: the new PennyLane template
+    """
+
+    def _function(params: dict = None, wires: list = None):
+        """Returns a PennyLane template created based on the input QuantumCircuit.
+        Warnings are created for each of the QuantumCircuit instructions that were
+        not incorporated in the PennyLane template.
+
+        Args:
+            params (dict): specifies the parameters that need to bound in the
+            QuantumCircuit
```

Comment:
(throughout): indentation?

Reply:
The `load` is a wrapper around `_function` here, would you suggest to have another indentation style in this case?

Reply:
I think this is a reference to the `Args:`; the line continuation should be indented, and there is no need to have a line break; i.e.
```
        Args:
            params (dict): specifies the parameters that need to bound in the
                QuantumCircuit
            wire_shift (list): the shift to be made with respect to the wires
```

Reply:
True, thanks!